src/plopp/core/utils.py

```diff
@@ -29,6 +29,8 @@ def coord_as_bin_edges(da: sc.DataArray, key: str, dim: str = None) -> sc.Variab
         x = sc.arange(dim, float(x.shape[0]), unit=x.unit)
     if da.meta.is_edges(key, dim=dim):
         return x
+    if x.dtype == int:
+        x = x.to(dtype='float64')
```

Comment:
It would be nice to keep axis tic labels as integers if the input is, does this influence this?

Reply:
Matplotlib convertes everything to float and also adds ticks in-between the integers if it decides it needs more ticks.
I don't think it's worth trying to fix.
![Screenshot at 2022-11-08 10-17-15](https://user-images.githubusercontent.com/39047984/200525014-386f4dea-c23f-4684-a306-fdc7af7604c5.png)